src/components/repo-kanban.jsx

```diff
@@ -77,6 +99,27 @@ class KanbanRepo extends Component {
   render() {
     const {columnData, cards, repoInfos} = this.props;
 
+    // Get review comments out of cards
+    const reviews = cards.map((card) => {
+      if (card.issue.pullRequest && card.issue.pullRequest.comments) {
+        let comments = card.issue.pullRequest.comments;
+        comments.forEach(comment => {
+          comment.repoOwner = card.repoOwner;
+          comment.repoName = card.repoName;
+          comment.number = card.number;
+        });
+        return comments;
```

Comment:
Hmmm, any reason for that? What's the benefit of using `map` here?

Reply:
Nevermind, it's inside map, didn't saw that.

But this will mutate the prop, it's considered bad practice in React, and IMO a map shouldn't mutate because it's normally for creating a new array. However, this will not cause problem since we know it's only adding new property that isn't used elsewhere.